# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/components/places/nsAnnoProtocolHandler.cpp
# Commit: 74f99033251c
# Full Hash: 74f99033251cd6c36eb6b0b5a4713eb6b5c793fa
# Author: Jonathan Kingston <jkt@mozilla.com>
# Date: 2019-02-13 09:48:26
# Regressor Bug: 1520868
# File Overlap Count: 1
# Description:
#   Bug 1520868 - Replacing AsyncOpen2 with AsyncOpen always r=valentin
#   
#   Replacing js and text occurences of asyncOpen2
#   Replacing open2 with open
#   
# ==============================================================================

diff -r 3378fd5f4f22 -r 74f99033251c toolkit/components/places/nsAnnoProtocolHandler.cpp
--- a/toolkit/components/places/nsAnnoProtocolHandler.cpp	Tue Feb 12 17:58:19 2019 +0100
+++ b/toolkit/components/places/nsAnnoProtocolHandler.cpp	Tue Feb 12 16:08:25 2019 +0000
@@ -161,7 +161,7 @@
       mListener->OnStopRequest(mChannel, nullptr, rv);
       return rv;
     }
-    return newChannel->AsyncOpen2(mListener);
+    return newChannel->AsyncOpen(mListener);
   }
 
  protected:
@@ -293,7 +293,7 @@
           nsresult rv = GetDefaultIcon(channel, getter_AddRefs(chan));
           NS_ENSURE_SUCCESS(rv, Err(rv));
 
-          rv = chan->AsyncOpen2(listener);
+          rv = chan->AsyncOpen(listener);
           NS_ENSURE_SUCCESS(rv, Err(rv));
 
           return RequestOrReason(chan.forget());